client/src/views/home.js

```diff
@@ -34,16 +34,16 @@ export const recentBlocks = ({ t, blocks, loading, ...S }) => homeLayout(
     <div className="blocks-table">
       <div className="blocks-table-row header">
         <div className="blocks-table-cell">{t`Height`}</div>
-        <div className="blocks-table-cell">{t`Timestamp`}</div>
+        <div className="blocks-table-cell">{`${t`Timestamp`} (${formatTimezone(new Date().getTime())})`}</div>
```

Comment:
`formatTimezone` expects a unix timestamp in seconds while `getTime()` returns it in milliseconds, so you should divide this by 1000. And you could also use the shorter `Date.now()`. So basically, `Date.now()/1000`.

But note that (as I mentioned on the [previous comment](https://github.com/Blockstream/esplora/pull/261#issuecomment-711434434)) using the timezone offset of the current timestamp is problematic because it might not match the timezone offset of the actual block timestamps.

Reply:
Changes have been made